frontend/src/components/Create.tsx

```diff
@@ -41,6 +44,10 @@ class Create extends React.Component<Props, State> {
     this.setState({ url: e.target.value });
   }
 
+  handleUserNameCreation(e: any) {
```

Comment:
This should probably be named `handleUsernameFieldChange` to match the naming convention of the other methods

Reply:
Refactored function name!